README.md

```diff
@@ -114,33 +114,38 @@ Add this to your `Cargo.toml`:
 
 ```toml
 [dependencies]
-rdkafka = "~0.21"
+rdkafka = { version = "0.22", features = ["cmake-build"] }
 ```
 
-This crate will compile librdkafka from sources and link it statically to your executable. To compile librdkafka you'll need:
+This crate will compile librdkafka from sources and link it statically to your
+executable. To compile librdkafka you'll need:
 
 * the GNU toolchain
 * GNU `make`
 * `pthreads`
-* `zlib`
-* `libssl-dev`: optional, *not* included by default (feature: `ssl`).
-* `libsasl2-dev`: optional, *not* included by default (feature: `sasl`).
+* `zlib`: optional, but included by default (feature: `libz`)
+* `cmake`: optional, *not* included by default (feature: `cmake-build`)
+* `libssl-dev`: optional, *not* included by default (feature: `ssl`)
+* `libsasl2-dev`: optional, *not* included by default (feature: `gssapi`)
+* `libzstd-dev`: optional, *not* included by default (feature: `zstd-pkg-config`)
 
-To enable ssl and sasl, use the `features` field in `Cargo.toml`. Example:
+Note that using the CMake build system, via the `cmake-build` feature, is
+**strongly** encouraged. The default build system has a [known
+issue](rdkafka-sys/README.md#known-issues) that can cause corrupted builds.
+
+By default a submodule with the librdkafka sources pinned to a specific commit
+will be used to compile and statically link the library. The `dynamic-linking`
+feature can be used to instead dynamically link rdkafka to the system's version
+of librdkafka. Example:
 
 ```toml
-[dependencies.rdkafka]
-version = "~0.21"
-features = ["ssl", "sasl"]
+[dependencies]
+rdkafka = { version = "0.22", features = ["cmake-build", "dynamic-linking"] }
```

Comment:
Is cmake-build necessary when using dynamic linking?

Reply:
D'oh, good catch, thanks!